libs/execution_context/ebpf_program.c

```diff
@@ -249,6 +249,9 @@ ebpf_program_create(ebpf_program_t** program)
         goto Done;
     }
 
+    ebpf_list_initialize(&local_program->links);
+    ebpf_lock_create(&local_program->links_lock);
```

Comment:
Where is this used?  Should this be part of a different PR?

Reply:
Driver verifier bug checks on this. I can move it to a different PR if needed, but it was blocking testing of this change (I ran with driver verifier enabled to catch bugs in my change and this was hit).

Reply:
I didn't see anything in the PR description about this.  So either move to another PR or update the PR description here.   Normally we try to avoid unrelated things being in the same PR, but not a hard rule, so up to you.

Reply:
Agreed. https://github.com/microsoft/ebpf-for-windows/pull/395

Reply:
This is being initialized in ebpf_program_initialize()